guide/14-deep-learning/FCN_MLModel_guide-final_draft.ipynb

```diff
@@ -0,0 +1,517 @@
+{
```

Comment:
<p>Once the explanatory variables and explanatory rasters are defined, we could prepare our TabularDataObject by using  <code>prepare_tabulardata</code> function.</p><p> </p><ul><li><code>input_layer</code>: feature layer or spatially enabled dataframe having the primary dataset</li><li><code>variable_predict</code>: : field name containing the y-variable from the input feature layer or dataframe</li><li><code>explanatory_variables</code>&nbsp;: list of the field names as 2-sized tuples containing the explanatory variables from the <code>input_layer</code> </li><li><code>explanatory_rasters</code>&nbsp;: list of the field names as 2-sized tuples containing the explanatory variables in multiple input rasters </li></ul><p> </p><pre class="ql-syntax" spellcheck="false">
data = prepare_tabulardata(input_feature, 
                           variable_predict,  
                           explanatory_variables=X, 
                           explanatory_rasters=explanatory_rasters)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>